W-19664579-salesforce-connector-update-kt

kevintroller opened the pull request:
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

kevintroller committed:
W-19664579-salesforce-connector-update-kt

dukesphere commented on salesforce/11.3/modules/ROOT/pages/salesforce-connector-reference.adoc:
```diff
@@ -390,7 +390,7 @@ Source that adds one or more `whereConditions` in the query calling the method.
 | Triggers Query a| String | Query used to filter the trigger results, for example, `Name='Filter'`. |  |
 | Since a| String |  Specify a date in the `YYYY-MM-DD hh:mm:ss` format, for example, `2017-03-17 16:30:40`. If this field is empty, this source retrieves the selected objects from the time the Mule app started. |  |
-| Primary Node Only a| Boolean |  Determines whether to execute this source on only the primary node when running Mule instances in a cluster. |  |
+| Primary Node Only a| Boolean |  Determines whether to execute this source on only the primary node when running Mule instances in a cluster. This parameter doesn't have any impact on non-cluster with multiple workers. |  |
```
On non-cluster what?

kevintroller committed:
W-19865331-salesforce-update-kt

dukesphere commented on salesforce/11.3/modules/ROOT/pages/salesforce-connector-processing-events.adoc:
```diff
@@ -10,6 +10,8 @@ NOTE: If you're adding a new connector for platform events and change data captu
 
 When deploying apps in a distributed environment such as a runtime cluster, the source must run on the primary node. Running the source on multiple nodes causes duplicate event consumption and object store data corruption for the data stored by Salesforce Connector.
 
+When deploying apps to a non-cluster mode with multiple workers, then the source runs on all the nodes which consumes duplicate events.
```
Should this be "deploying in non-cluster mode"? "deploying to" sound like we're deploying to the mode.

kevintroller committed:
W-19865331-salesforce-update-kt

kevintroller committed:
Merge branch 'latest' into W-19865331-salesforce-update-kt

kevintroller committed:
Update Salesforce connector docs for versions 11.0-11.3

- Add statement about non-cluster mode with multiple workers consuming duplicate events in processing-events.adoc
- Update Primary Node Only parameter description to clarify no impact on non-cluster mode in reference.adoc

kevintroller committed:
Merge branch 'latest' into W-19865331-salesforce-update-kt
